android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/report/measure/MeasureReportConfig.kt

```diff
@@ -25,4 +25,10 @@ data class MeasureReportConfig(
   val description: String = "",
   val url: String = "",
   val module: String = "",
+  val type: MeasureReportConfigType = MeasureReportConfigType.PATIENT,
 )
+
+enum class MeasureReportConfigType {
```

Comment:
can you please propose a way to refactor this that remove this type and the conditional associated with it from the hard coding?

Reply:
Sure, I have some changes to the way indicators are being handled so it would be rendered in a generic way i.e. For Group level resources if type is subject show counts from MeasureReport.contained > Observation The work is under progress and hoping to completely remove the enum by next week.